nnfabrik/templates/transfer/recipes.py

```diff
@@ -0,0 +1,169 @@
+from typing import Dict
+
+from nnfabrik.main import *
+
+
+class Recipe(dj.Lookup):
```

Comment:
Not sure how I feel about the name here. For me the name of a table should describe what a single entry in that table represents. Here I expect each entry in the table to represent a recipe it represents a step in a potential recipe. So something like `TransferStep` might be a more accurate name.

Reply:
I disagree. The table defines the how transfer table is supposed to be populated. I find `Recipe` fits well.

Reply:
Okay, fair. I was just a bit confused at first because  I would have expected that each entry in the `Recipe` table represents a complete transfer recipe, i.e. a sequence of individual transfers a particular set of trained models would go through. Currently for me the entries in the `Recipe` table represent a single step in such a recipe...